http/src/main/resources/init-resources/startup.sh

```diff
@@ -64,6 +65,34 @@ export WELDER_MEM_LIMIT=$(welderMemLimit)
 export MEM_LIMIT=$(memLimit)
 export USE_GCE_STARTUP_SCRIPT=$(useGceStartupScript)
 
+function failScriptIfError() {
+  if [ $EXIT_CODE -ne 0 ]; then
+    echo "Fail to docker-compose start welder ${EXIT_CODE}. Output is saved to ${JUPYTER_START_USER_SCRIPT_OUTPUT_URI}"
```

Comment:
I thought `JUPYTER_START_USER_SCRIPT_OUTPUT_URI` was just intended for the "restart" user script -- do we want to overload it for other purposes like starting welder?

Also the log message here mentions welder but this function is used for other purposes..

Reply:
yea..I realize the original purpose of the output is only for user input startup script..but we actually do care if startup script failed anywhere tho, and we should exit startup instead of currently things just stuck in starting

Reply:
Yeah true.. I guess users probably only look at the output log in case of errors, so it's useful to have more types of errors in there.